compiler/control/CompileMethod.cpp

```diff
@@ -80,7 +80,7 @@ writePerfToolEntry(void *start, uint32_t size, const char *name)
       static const int maxPerfFilenameSize = 15 + sizeof(jvmPid)* 3; // "/tmp/perf-%ld.map"
       char perfFilename[maxPerfFilenameSize] = { 0 };
 
-      int numCharsWritten = snprintf(perfFilename, maxPerfFilenameSize, "/tmp/perf-%ld.map", jvmPid);
+      int numCharsWritten = snprintf(perfFilename, maxPerfFilenameSize, "/tmp/perf-%ld.map", (long)jvmPid);
```

Comment:
In the compiler code, we prefer using `int*_t` rather than `int`, `long`, etc. Would it be possible to instead have:

```suggestion
      int numCharsWritten = snprintf(perfFilename, maxPerfFilenameSize, "/tmp/perf-%" OMR_PRId64 ".map", (int64_t)jvmPid);
```

?

Reply:
OK, I'll fix this, and change the cast to reinterpret_cast.

Reply:
fixed, and I changed the cast to use `static_cast`.